packages/headless/src/api/analytics/analytics.ts

```diff
@@ -139,6 +141,14 @@ export class AnalyticsProvider implements SearchPageClientProvider {
   }
 }
 
+export class CaseAssistAnalyticsProvider implements CaseAssistClientProvider {
+  constructor(private state: StateNeededByCaseAssistAnalytics) {}
+
+  public getOriginLevel1() {
```

Comment:
Isn't the originLevel1 and searchHub two sides for the same coin? and the names `originLevel1` and `searchHub` can be interchangeable?
Also Changing the name of this method will oblige us to also change the name of this method in the `CaseAssistClientProvider` interface in coveo.analytics.js which I'm afraid  will introduce a breaking change to this library 😕

Reply:
Ah darn. Well to avoid the breaking change let's leave it then. But while the two terms are used to indicate the same thing, I don't think we should regard them as interchangeable. We use the searchHub as the value for originLevel1 but they have different meanings. In the platform and the doc we don't say "assign your query pipeline to a given originLevel1". Do you know what I mean? The concept of searchHub is already often misunderstood so giving it an alias can be even more confusing for users. But that's just my feeling and the breaking change argument is strong so no worries.